.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,156 @@
+id:
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description:
+      frequencies:
+      - hourly:
+          hour: 1
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: 'Needs: Author Feedback'
+      - hasLabel:
+          label: 'Status: No Recent Activity'
+      - noActivitySince:
+          days: 3
+      - isNotLabeledWith:
+          label: Service Bug
+      actions:
+      - closeIssue
+    - description:
+      frequencies:
+      - hourly:
+          hour: 1
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: 'Needs: Author Feedback'
+      - noActivitySince:
+          days: 4
+      - isNotLabeledWith:
+          label: 'Status: No Recent Activity'
+      - isNotLabeledWith:
+          label: Service Bug
+      actions:
+      - addLabel:
+          label: 'Status: No Recent Activity'
+      - addReply:
+          reply: This issue has been automatically marked as stale because it has been marked as requiring author feedback but has not had any activity for **4 days**. It will be closed if no further activity occurs **within 3 days of this comment**.
+    - description:
+      frequencies:
+      - hourly:
+          hour: 1
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: duplicate
+      - noActivitySince:
+          days: 1
+      actions:
+      - addReply:
+          reply: This issue has been marked as duplicate and has not had any activity for **1 day**. It will be closed for housekeeping purposes.
+      - closeIssue
+    - description:
+      frequencies:
+      - hourly:
+          hour: 1
+      filters:
+      - isOpen
+      - hasLabel:
+          label: 'API: Graph'
+      - noActivitySince:
+          days: 14
+      actions:
+      - addLabel:
+          label: Service issue
+      - addReply:
+          reply: >-
+            Hello @${issueAuthor}
+
+
+            Thank you for reporting your concern. Unfortunately, we in the Graph Explorer team do not have any ownership of the APIs that are causing you issues. If you report this issue in the Microsoft Q&A forum, it will get routed to the appropriate team for them to triage.
```

Comment:
looks like it is copied from the GE repo. Is it applicable here?

Reply:
Nope. I started copying the wrong file.